src/main/java/core/basesyntax/model/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import core.basesyntax.Machine;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Common Excavator, You can do it");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Ohh, Excavator  try again");
```

Comment:
The stopWork method message 'Ohh, Excavator  try again' is unclear. It should clearly indicate that the Excavator is stopping its work, for example, 'Excavator has stopped digging'.